src/Formatter/MarkdownFormatter.php

```diff
@@ -0,0 +1,60 @@
+<?php
+declare(strict_types=1);
+
+namespace Roave\ApiCompare\Formatter;
+
+use Roave\ApiCompare\Change;
+use Roave\ApiCompare\Changes;
+
+final class MarkdownFormatter implements OutputFormatter
+{
+    /**
+     * @var string
+     */
+    private $outputFilename;
+
+    public function __construct(string $outputFilename)
+    {
+        $this->outputFilename = $outputFilename;
+    }
+
+    public function write(Changes $changes) : void
+    {
+        $arrayOfChanges = $changes->getIterator()->getArrayCopy();
+
+        file_put_contents(
```

Comment:
As of above: to be avoided. Let's rely on standard piping and output redirection techniques instead.